display_list/display_list_image_filter_unittests.cc

```diff
@@ -43,6 +43,40 @@ TEST(DisplayListImageFilter, FromSkiaBlurImageFilter) {
 
   // We cannot recapture the blur parameters from an SkBlurImageFilter
   ASSERT_EQ(filter->asBlur(), nullptr);
+  ASSERT_EQ(filter->asDilate(), nullptr);
+  ASSERT_EQ(filter->asErode(), nullptr);
+  ASSERT_EQ(filter->asMatrix(), nullptr);
+  ASSERT_EQ(filter->asCompose(), nullptr);
+  ASSERT_EQ(filter->asColorFilter(), nullptr);
+}
+
+TEST(DisplayListImageFilter, FromSkiaDilateImageFilter) {
+  sk_sp<SkImageFilter> sk_image_filter =
+      SkImageFilters::Dilate(5.0, 5.0, nullptr);
+  std::shared_ptr<DlImageFilter> filter = DlImageFilter::From(sk_image_filter);
+
+  ASSERT_EQ(filter->type(), DlImageFilterType::kUnknown);
+
+  // We cannot recapture the dilate parameters from an SkDilateImageFilter
+  ASSERT_EQ(filter->asDilate(), nullptr);
```

Comment:
There are too calls verifying asDilate and non testing asBlur. I think some cutting/pasting went awry here.